### DISCUSSION TITLE
Video Review needed

### DISCUSSION THREAD

**Discussion by rainthree** [Interlisp/medley#1318] [General]
Video:
https://www.youtube.com/watch?v=39lBLohuabM

Transcript:
https://web.archive.org/web/20230830073854/https://cl1p.net/interlisp1txtv2

Corrections are needed, and tips are more than welcome 
Thank you!

**Comment by nbriggs** [comment]
As far as tips on manipulating files - I wouldn't encourage editing the fileCOMS as a variable to create a new file.  If you've defined a new function and you run (FILES?) it will ask you if you want to say where the new function should go, and let you type the base file name, and will confirm creating a new file (the in-memory info about the file, e.g., the fileCOMS variable) which you can then dump with MAKEFILE or CLEANUP or add special things that aren't functions/variables/etc.  with (DC file) (like DF/DV but for COMS instead of FNS/VARS)

![Screen Shot 2023-08-31 at 11 11 19 AM](https://github.com/Interlisp/medley/assets/8138841/3f20f980-2633-4f74-bfe6-4d4ef21c8a36)
